src/KingSystem/ActorSystem/actInfoData.h

```diff
@@ -74,6 +74,10 @@ class InfoData {
         sead::SafeArray<sead::SafeString, 16> times;
         s32 num_weathers;
         sead::SafeArray<sead::SafeString, 16> weathers;
+
+        bool containsCurrentTimeOrWeather(sead::Vector3f* a2);
```

Comment:
These should probably take `const sead::Vector3f& pos` considering the position must **not** be null and the vector is never modified